integration-tests/typings/embrace.d.ts

```diff
@@ -27,17 +27,64 @@ interface EmbraceSpanData {
   attributes: EmbraceSpanAttribute[];
 }
 
+interface EmbraceLogData {
+  trace_id: string;
+  severity_number: number;
+  severity_text: string;
+  body: string;
+  time_unix_nano: number;
+  status: "Unset" | "Ok" | "Error";
+  events: EmbraceSpanEvent[];
+  attributes: EmbraceSpanAttribute[];
+}
+
+interface ParsedLogPayload {
+  resource?: EmbracePayloadResource;
```

Comment:
I don't think we need resource + metadata here if it's on the EmbracePayload

Reply:
ok